internal/runners/packages/searchView.go

```diff
@@ -0,0 +1,245 @@
+	keyWebsite     = locale.Tl("search_website", "Website")
+	keyLicense     = locale.Tl("search_license", "License")
+	keyVersions    = locale.Tl("search_versions", "Versions")
+	keyVulns       = locale.Tl("search_vulnerabilities", "Vulnerabilities (CVEs)")
+
+	keys = []string{
+		keyName,
+		keyDescription,
+		keyWebsite,
+		keyLicense,
+		keyVersions,
+		keyVulns,
+	}
+)
+
+const (
+	leftPad        = 2
+	verticalMargin = 7
+	scrollUp       = "up"
+	scrollDown     = "down"
+)
+
+type view struct {
+	width         int
+	height        int
+	content       string
+	searchResults *structuredSearchResults
+	ready         bool
+	viewport      viewport.Model
+}
+
+func NewView(results *structuredSearchResults, out output.Outputer) (*view, error) {
+	outFD, ok := out.Config().OutWriterFD()
+	if !ok {
+		logging.Error("Could not get output writer file descriptor, falling back to stdout")
+		outFD = os.Stdout.Fd()
+	}
+
+	width, height, err := term.GetSize(int(outFD))
+	if err != nil {
+		return nil, errs.Wrap(err, "Could not get terminal size")
+	}
+
+	return &view{
+		width:         width,
+		height:        height,
+		searchResults: results,
+	}, nil
+}
+
+func (v *view) Init() tea.Cmd {
+	return nil
+}
+
+func (v *view) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	var (
+		cmd  tea.Cmd
+		cmds []tea.Cmd
+	)
+
+	switch msg := msg.(type) {
+	case tea.MouseMsg:
+		switch msg.Button {
+		case tea.MouseButtonWheelUp:
+			v.viewport.LineUp(3)
+			return v, nil
+		case tea.MouseButtonWheelDown:
+			v.viewport.LineDown(3)
+			return v, nil
+		}
+	case tea.KeyMsg:
+		stringMsg := strings.ToLower(msg.String())
+		switch stringMsg {
+		case "q", "ctrl+c":
+			return v, tea.Quit
+		case "up":
+			v.viewport.LineUp(1)
+			return v, nil
+		case "down":
+			v.viewport.LineDown(1)
+			return v, nil
+		case "pgup":
+			v.viewport.LineUp(v.height - verticalMargin)
+			return v, nil
+		case "pgdown":
+			v.viewport.LineDown(v.height - verticalMargin)
+			return v, nil
+		}
+	case tea.WindowSizeMsg:
+		if !v.ready {
+			// Keep the searching message and command in view
+			v.viewport = viewport.New(msg.Width, msg.Height-verticalMargin)
+			v.content = v.processContent()
+			v.viewport.SetContent(v.processContent())
+			v.ready = true
+		} else {
+			v.width = msg.Width
+			v.height = msg.Height
+		}
+	}
+
+	v.viewport, cmd = v.viewport.Update(msg)
+	cmds = append(cmds, cmd)
+
+	return v, tea.Batch(cmds...)
+}
+
+func (v *view) View() string {
+	return v.viewport.View() + "\n\n" + v.footerView()
+}
+
+func (v *view) processContent() string {
+	maxKeyLength := 0
+	for _, key := range keys {
+		renderedKey := colorize.StyleBold.Render(key)
+		if len(renderedKey) > maxKeyLength {
+			maxKeyLength = len(renderedKey) + 2
+		}
+	}
+
+	doc := strings.Builder{}
+	for _, pkg := range v.searchResults.Results {
+		if pkg.Name != "" {
+			doc.WriteString(formatRow(colorize.StyleBold.Render(keyName), pkg.Name, maxKeyLength, v.width))
```

Comment:
I've updated the mock to make the keys greyed out (ie. the disabled style).

Playing with the live results the emphasis is too much on the keys at the moment. Making them grey should help with de-emphasising the keys and focussing the attention on the actual results. This'll also help with making the name stand out.